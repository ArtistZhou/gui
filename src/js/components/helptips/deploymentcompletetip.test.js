import React from 'react';
import renderer from 'react-test-renderer';
import DeploymentCompleteTip from './deploymentcompletetip';

it('renders correctly', () => {
  const tree = renderer.create(<DeploymentCompleteTip />).toJSON();
  expect(tree).toMatchSnapshot();
});
