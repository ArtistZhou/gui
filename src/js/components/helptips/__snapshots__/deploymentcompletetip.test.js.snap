// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <p>
    Fantastic! You completed your first deployment!
  </p>
  <p>
    Your deployment is finished and your device is now running the updated software!
  </p>
  <div
    className="flexbox centered"
  >
    <div
      className="loaderContainer"
    >
      <div
        className="loader"
      >
        <span
          className="dot dot_1"
        />
        <span
          className="dot dot_2"
        />
        <span
          className="dot dot_3"
        />
        <span
          className="dot dot_4"
        />
      </div>
    </div>
  </div>
  <p>
    and you should see the demo web application actually being run on the device.
  </p>
  <p>
    NOTE: if you have local network restrictions, you may need to check them if you have difficulty loading the page.
  </p>
  <div
    className="loaderContainer"
  >
    <div
      className="loader"
    >
      <span
        className="dot dot_1"
      />
      <span
        className="dot dot_2"
      />
      <span
        className="dot dot_3"
      />
      <span
        className="dot dot_4"
      />
    </div>
  </div>
</div>
`;
